db/models/Order.js

```diff
@@ -0,0 +1,37 @@
+const db = require("../db");
+const { Sequelize } = db;
+
+const Order = db.define("order", {
+    id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true
+    },
+    status: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        validate: {
+            notEmpty: {
+                args: true,
+                msg: "Order must have a status"
+            },
+            isIn: {
+                args: [["purchased", "shipped"]],
+                msg: "Order status must be either 'purchased' or 'shipped'"
+            }
+        }
+    },
+    totalAmount: {
```

Comment:
This looks good, maybe later we can write a function to get the amount in the backend. For now, we can calculate the total amount on the front end and pass it to the axios call.